x-pack/legacy/plugins/apm/cypress/cypress/integration/apm.feature

```diff
@@ -0,0 +1,9 @@
+Feature: APM
+
+  Background:
+    Given a user browses the APM UI application
+    When the user inspects the opbeans-go service
```

Comment:
The `When` clause it really important, because it's the one triggering the action we want to verify its behavior.